{
  "dataType": "CVE_RECORD",
  "dataVersion": "5.1",
  "cveMetadata": {
    "total_count": 1,
    "last_updated": "2025-09-26T18:34:31.025306",
    "severity_distribution": {
      "critical": 0,
      "high": 1,
      "medium": 0,
      "low": 0,
      "none": 0
    }
  },
  "cves": [
    {
      "id": "CVE-2025-10951",
      "publishedDate": "2025-09-25T15:02:06.416Z",
      "lastModifiedDate": "2025-09-26T18:07:15.542Z",
      "description": "A vulnerability was identified in geyang ml-logger up to acf255bade5be6ad88d90735c8367b28cbe3a743. Affected by this vulnerability is the function log_handler of the file ml_logger/server.py. Such manipulation of the argument File leads to path traversal. It is possible to launch the attack remotely. The exploit is publicly available and might be used. This product takes the approach of rolling releases to provide continious delivery. Therefore, version details for affected and updated releases are not available.",
      "severity": 7.3,
      "references": [
        {
          "url": "https://vuldb.com/?id.325821",
          "type": "reference"
        },
        {
          "url": "https://vuldb.com/?ctiid.325821",
          "type": "reference"
        },
        {
          "url": "https://vuldb.com/?submit.652462",
          "type": "reference"
        },
        {
          "url": "https://github.com/geyang/ml-logger/issues/73",
          "type": "poc"
        }
      ],
      "affected": [
        {
          "vendor": "geyang",
          "product": "ml-logger",
          "versions": [
            {
              "version": "acf255bade5be6ad88d90735c8367b28cbe3a743",
              "status": "affected"
            }
          ]
        }
      ],
      "problemType": [
        "Path Traversal"
      ],
      "fix_suggestion": "无法生成修复建议"
    }
  ]
}